tests/www/LoginIntegrationTest.php

```diff
@@ -282,4 +353,60 @@ private function authenticate()
         );
         $this->assertEquals(200, $resp['code']);
     }
+
+    /**
+     * TODO: migrate into BuiltInServer
```

Comment:
If you migrate, consider using the BuiltInServer from `simplesamlphp/simplesamlphp-test-framework` to be future-proof